specification/quota/resource-manager/Microsoft.Quota/preview/2019-07-19-preview/quota.json

```diff
@@ -0,0 +1,609 @@
+            "schema": {
+              "$ref": "#/definitions/ExceptionResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Quota/providers/{providerId}/locations/{locationId}/quotaRequests/{requestName}": {
+      "put": {
+        "tags": [
+          "QuotaRequests"
+        ],
+        "summary": "Submits a Quota Request for a resource provider at the specified location.",
+        "description": "Submits Quota change request for a resource provider for the specified location. To use, first make a request to the /usages API to get quota information. This information consists of a list of resources and information regarding those resources. For all the resources in that list which require an update to their quotas, update their limit fields in the response from the /usages to their new values. Then, submit this updated JSON object to this quota request API. This will update the quotas to the values specified.",
+        "operationId": "Quota_Request_Submit",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ProviderIdParameter"
+          },
+          {
+            "$ref": "#/parameters/LocationIdParameter"
+          },
+          {
+            "$ref": "#/parameters/QuotaRequestNameParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "createQuotaRequests",
+            "in": "body",
+            "description": "Quota requests payload.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CreateGenericQuotaRequestParameters"
+            }
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted. Quota requests will be created asynchronously. A request id for each changed resource is returned in the same order as the order in which they were provided.",
+            "schema": {
+              "$ref": "#/definitions/QuotaRequestResponseStatus"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ExceptionResponse"
+            }
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "QuotaRequests"
+        ],
+        "summary": "For the specified location, gets the Quota request status by request name provided during the quota request creation for the specific resources under the subscription.",
+        "description": "Gets the QuotaRequest details and status by the quota Request Name for the resources for the resource  provider at a specific location.",
+        "operationId": "QuotaRequests_Status_Check",
```

Comment:
The summary is confusing, sounds like only one item is returned but the model indicates otherwise, can you please clarify?  If it's singular then it should be `QuotaRequests_Get`.

Reply:
That's right, only one item is returned.
Updated the response type to quotaRequestDetail. Please check the previous comment.
Updated OP-ID to QuotaRequests_Get